consensus/state.go

```diff
@@ -1100,6 +1103,19 @@ func (cs *State) defaultDecideProposal(height int64, round int32) {
 	} else if !cs.replayMode {
 		cs.Logger.Error("enterPropose: Error signing proposal", "height", height, "round", round, "err", err)
 	}
+
+	// post data to ipfs
+	// TODO(evan): don't hard code context and timeout
```

Comment:
Let's track minor todos like this this in a follup issue before we merge.

Reply:
It's not about management. But from my experience todos can be forgotten and hence it is good to capture them somewhere more visible. It does not need to be one issue per todo. We can collect all minor things like this in one issue (see for instance: #179).

Reply:
I've tracked these todos in #204 in order to prevent github calling me out on twitter. :smile: 
https://twitter.com/github/status/1367885997527171073?s=20